packages/pn-personafisica-webapp/src/components/Contacts/ContactsSummaryCards.tsx

```diff
@@ -89,26 +92,19 @@ const ContactsSummaryCard: React.FC<ContactsSummaryCardProps> = ({
       sx={{ width: { xs: '100%', lg: '185px' } }}
       data-testid={isCourtesyCard ? 'courtesyContactsCard' : 'legalContactsCard'}
     >
-      <CardActionArea
-        onClick={goToSection}
-        sx={{ p: 2 }}
-        aria-label={t(title)}
-        aria-description={getDescription()}
-      >
+      <CardActionArea onClick={goToSection} sx={{ p: 2 }}>
         {getIcon()}
         <Typography
           variant="body2"
           fontWeight={600}
           data-testid="cardTitle"
-          aria-hidden
```

Comment:
this should not be removed. why did you remove it?

Reply:
on windows the card are not read

Reply:
do I restore it?

Reply:
with aria-hidden the text 'A valore legale' and 'Di cortesia' are not read by windows voice assistant.